### ISSUE TITLE
PMID:22267499 Ypa1, Ypa2   SIN signalling

### ISSUE THREAD

**Issue by ValWood** [pombase/curation#3439]
Ypa1, Ypa2  
GO:0023057  negative regulation of SIN signalling
GO:0008160 protein tyrosine phosphatase activator activity EXP
PMID:22267499 prioritise

**Comment by ValWood** [comment]
in progress curs/291d614ef0c90bc4

**Comment by ValWood** [comment]
@PCarme a heads up that I changed ypa2 to regulation of SIN signalling,  so it should fit into your model, I'm not sure how?

**Comment by PCarme** [comment]
Probably through the SIP complex which contains other PP2A subunits and dephosphorylates Cdc11 via Ppa3. I will have to search for the missing link there.

**Comment by PCarme** [comment]
I just noticed that this paper is listed in the literature category on the ppa3 gene page 
![image](https://github.com/pombase/curation/assets/163283897/d7613d02-a71e-49d2-86b6-6623c4ec10a5)

although it contains no annotations regarding ppa3, which is only mentioned in the introduction.
![image](https://github.com/pombase/curation/assets/163283897/33a594ad-3e01-4a3b-ba76-717ecd9288c0)

**Comment by ValWood** [comment]
We only began to do this gene extraction from publications recently. I guess this is because we get this info from the PubMed metadata (I gues ppa3 must be included in the keywords).
We did decide (although I can't remember if a ticket was opened), that once a publication is curated in PomBAse, we could default to the gene list from Canto  (which would remove this type of anomaly, in addition to the problem when  a gene has conflicting names.

I will open a ticket for this if there isn't one. Opening this as a reminder.

**Comment by ValWood** [comment]
Although it's a bit weird because 

https://pubmed.ncbi.nlm.nih.gov/?cmd=Search&term=(SPAC22H10.04+OR+ppa3+OR+ppg1)+AND+(pombe+OR+fission+yeast)

does not retrieve ANY fission yeast ppa3 publications???

**Comment by PCarme** [comment]
This is strange. It looks like the searchbar has a problem with ppa3 in pombe.

**Comment by PCarme** [comment]
And searching for ppa3 AND yeast only finds this paper about a plant protein (with some data with expression in yeast). 
https://pubmed.ncbi.nlm.nih.gov/29691313/

**Comment by PCarme** [comment]
> Are there any true positives for ppa3?
> Part of the problem might be that people never call it ppa3 they call it PPA2 because its PPA2 catalytic subunit.
> I can't find a paper the uses ppa3.

You're right, I can't find a single pombe paper with ppa3 in either the title or abstract.

**Comment by ValWood** [comment]
I think this can close, other than 2 small things:

i) ypa2 is the phosphatase activator (although I suspect it's actually involved in substrate recognition, in which case we should curate as a substrate adaptor , if there is evidence that this is the mechanism for this family).

Otherwise, this looks odd:

<img width="734" height="217" alt="Image" src="https://github.com/user-attachments/assets/d58936c7-f899-459b-866d-37950f12c46c" />

because the direct input for the GO term "phosphatase regulator" can only be a phosphatase.

another option, if this information cannot be found is to make it causa;;y upstream.

ii) Is it  possible to add the direction of regulation?

**Comment by PCarme** [comment]
> i) ypa2 is the phosphatase activator (although I suspect it's actually involved in substrate recognition, in which case we should curate as a substrate adaptor , if there is evidence that this is the mechanism for this family).
> 
> Otherwise, this looks odd:
> 
> <img alt="Image" width="734" height="217" src="https://private-user-images.githubusercontent.com/7359272/530768335-d58936c7-f899-459b-866d-37950f12c46c.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Njc3ODY2MzEsIm5iZiI6MTc2Nzc4NjMzMSwicGF0aCI6Ii83MzU5MjcyLzUzMDc2ODMzNS1kNTg5MzZjNy1mODk5LTQ1OWItODY2ZC0zNzk1MGYxMmM0NmMucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI2MDEwNyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNjAxMDdUMTE0NTMxWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NWYyNjc0YjNmYmJkY2E2NmY1NTE0NjA4YWVkMzZlYWE1YjNlNGUzZGZiOTg4NDAzOWIzNTIxZTA2MmQ4YTdmOCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.Ad77fp15R2VwalngMsa4LUXHJbWd_9_SQug6tNFmi4I">
> because the direct input for the GO term "phosphatase regulator" can only be a phosphatase.

I don't see the problem here. Ypa2 is annotated as a phosphatase activator activating the phosphatase activity of the FAR/SIN/STRIPAK complex (a PP2A type of complex).

> ii) Is it possible to add the direction of regulation?

Do you mean the direction in which Ypa2 regulates the SIN ? Because I think it already is clear from the model: it activates a phosphatase complex which inhibits Cdc11, the main scaffold of the SIN, so it negatively regulates the SIN

**Comment by ValWood** [comment]
ignor, i forgot FAR/SIN/STRIPAK was a phosphatase
